contrib/gitian-descriptors/gitian-linux.yml

```diff
@@ -43,6 +43,7 @@ script: |
   HOST_LDFLAGS=-static-libstdc++
 
   export QT_RCC_TEST=1
+  export QT_RCC_SOURCE_DATE_OVERRIDE=1
```

Comment:
QT_RCC_TEST was the variable for this in the past. If QT_RCC_SOURCE_DATE_OVERRIDE is the new one, let's just replace QT_RCC_TEST. I assume there's no need for both.

Same goes for the other descriptors as well.

Reply:
removed `QT_RCC_TEST=1` from gitian descriptors and didn't notice any break in gitian results determinism.